src/GetMarkdownMetadataCommand.cs

```diff
@@ -34,6 +34,11 @@ public sealed class GetMarkdownMetadataCommand : PSCmdlet
 
         #endregion
 
+        protected override void BeginProcessing()
+        {
+            Directory.SetCurrentDirectory(this.SessionState.Path.CurrentLocation.Path);
```

Comment:
Instead of doing this, you should use the PowerShell Path APIs to resolve the user input for `-Path`.

Reply:
The problem with that is if the path does not exist, it throws. For example, the output folder does not exist most of the times.

```
[System.Management.Automation.SessionState]::new().Path.GetResolvedProviderPathFromPSPath("d:\temp\..\doesnotexist", [ref] $x)
MethodInvocationException: Exception calling "GetResolvedProviderPathFromPSPath" with "2" argument(s): "Cannot find path 'D:\doesnotexist' because it does not exist."
```

It seems to be resolving it according to the error message, but can we rely on it? There is no output of course.

Reply:
I will be using UnresolvedPath..